.github/workflows/python.yml

```diff
@@ -9,47 +9,66 @@ on:
 
 jobs:
   build:
-    runs-on: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}-latest
     strategy:
+      fail-fast: False
       matrix:
-        os: [windows-latest, ubuntu-latest, macos-latest]
-        python-version: [3.9]
+        os: [windows, ubuntu, macos]
+        py: ["3.10"]
```

Comment:
Sticking to `python-version` is probably clearer.

Reply:
Done!